lib/launchers/o_c_m_cluster.rb

```diff
@@ -263,10 +263,17 @@ def create_ocpinfo_file(osd_name, dir, filename = 'OCPINFO.yml')
     def wait_for_osd(osd_name)
       loop do
         osd_status = get_value(osd_name, "state")
-        ocp_version = get_value(osd_name, "openshift_version")
+        ocp_version = "NONE"
```

Comment:
Shall we put this line outside of the loop ?

Reply:
Hi @liangxia I have put it outside of the loop.